opencti-platform/opencti-graphql/src/database/redis.ts

```diff
@@ -949,6 +949,42 @@ export const redisSetExclusionListCache = async (cache: ExclusionListCacheItem[]
 
 // endregion - exclusion list cache handling
 
+// region - forgot password handling
+
+export const OTP_TTL = conf.get('app:forgot_password:otp_ttl');
+
+export const redisSetForgotPasswordOtp = async (
+  id: string,
```

Comment:
Not a blocker, but I think the id here is the transaction id right ?
```suggestion
  transactionId: string,
```